.eslintrc.js

```diff
@@ -39,6 +39,9 @@ module.exports = {
         // Just do not agree with this one
         'no-negated-condition': 'off',
 
+        'formatjs/enforce-placeholders': 'error',
+        // 'formatjs/no-literal-string-in-jsx': 'error',
```

Comment:
Keeping this off for right now. Will make a stand alone PR to fix all of these while turning it on.